### PR TITLE
Run A Node Grants round

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Adds the new Run A Node Grants application.

Preview link: https://deploy-preview-291--ecosystem-support.netlify.app/run-a-node-grants

### CODE REVIEW

File: src/pages/run-a-node-grants/index.tsx [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+                  <PageSection mb={6}>Why Run a Node?</PageSection>
+
+                  <PageText mb={6}>
+                    We want you to be creative and demonstrate why you want to run a node for the
+                    Ethereum ecosystem. This list is to inspire you:
+                  </PageText>
+
+                  <List>
+                    <ListItem>
+                      Based in a remote location and want to test the node performance
```
Comment (corwintines) [inline src/pages/run-a-node-grants/index.tsx:130]:
```suggestion
                      Based in a remote location and want to test the node performance.
```
Just adding period to keep consistent with rest of list

File: src/pages/run-a-node-grants/index.tsx [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+                    </ListItem>
+                    <ListItem>
+                      Must have a fast, stable internet connection with high data thresholds.{' '}
+                      <Link
+                        fontWeight={700}
+                        color='brand.orange.100'
+                        href='https://www.speedtest.net/'
+                        isExternal
+                      >
+                        Test your internet connection
```
Comment (corwintines) [inline src/pages/run-a-node-grants/index.tsx:185]:
```suggestion
                        Test your internet connection.
```

Adding period for consistency with rest of list

File: src/pages/run-a-node-grants/index.tsx [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+                      </Link>
+                    </ListItem>
+                  </List>
+                </section>
+
+                <section id='next-steps-and-support' ref={ref7}>
+                  <PageSection mb={6}>Next steps and support</PageSection>
+
+                  <PageText mb={6}>
+                    For general support questions about your submission, please email PGR INBOX or{' '}
```
Comment (corwintines) [inline src/pages/run-a-node-grants/index.tsx:305]: Whats `PGR INBOX`?

File: src/pages/run-a-node-grants/index.tsx [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+                  <PageText mb={6}>
+                    Evaluation window: <b>June 20 - July 15</b>
+                  </PageText>
+
+                  <PageText mb={6}>
+                    Expect final decisions by <b>July 15</b>
+                  </PageText>
+
+                  <PageText mb={6}>
+                    Node launch and ENR submission deadline: <b>September 1, 2023</b>
```
[the remark this answers is not in the document]
Comment (monetmignon) [reply src/pages/run-a-node-grants/index.tsx:215]: let's go with `Ethereum Node Records (ENR)` ? ty!

File: src/pages/run-a-node-grants/index.tsx (hunk 3 above)
Comment (monetmignon) [reply src/pages/run-a-node-grants/index.tsx:305]: lol agreed, it's odd. remove that text please. so just `For general support questions about your submission, please email grant-rounds@ethereum.org`

File: src/pages/run-a-node-grants/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (corwintines) [reply src/pages/run-a-node-grants/index.tsx:215]:
```suggestion
                    Node launch and Ethereum Node Records (ENR) submission deadline: <b>September 1, 2023</b>
```

Changed to above suggestion

**Review by corwintines — APPROVED** [review]

**Pull request merged**